lib/swagcov/coverage.rb

```diff
@@ -31,8 +31,10 @@ def report
     attr_reader :dotfile
 
     def collect_coverage
+      openapi_files = ::Swagcov::OpenapiFiles.new(filepaths: dotfile.doc_paths)
+
       @routes.each do |route|
-        path = route.path.spec.to_s.sub(/\(\.:format\)$/, "")
+        path = route.path.spec.to_s.chomp("(.:format)")
```

Comment:
@jaydorsey since we came across this line in a recent chat...this cleans it up a bit 😉